client/locations-list-map.js

```diff
@@ -317,7 +317,16 @@ function multilineStringToNodes(input) {
 
 function createMakerMarkerContent(entry, separator) {
   // Text to go into InfoWindow
-  const contentTags = [ce('h5', separator ? 'separator' : null, ctn(entry.name))];
+  const contentTags = [];
```

Comment:
it looks like this is code from @ngottlieb previous pr, do you know how that got in here?

Reply:
Ahh I got completely lost in version control and had some old commits. I tried to commit the newest copy of location-list.js but there must have been some mixup. I'll revert it to the deleted line.

Reply:
Yeah I checked the location-list.js and there are so many changes that I didn't intend to introduce. I only wanted to change setBanana and locale, do you know how I can fix this?